### PR TITLE
Add Tekton Result build configuration for HACBS 

### PR DESCRIPTION
**Pull request by avinal**
## Changes 
- PR to add pull request/push PipelineRun configs to tekton results: https://github.com/openshift-pipelines/tektoncd-results/pull/5
- Jira issue: [PLNSRVCE-548](https://issues.redhat.com//browse/PLNSRVCE-548)

Signed-off-by: Avinal Kumar <avinal@redhat.com>

### CODE REVIEW

File: components/build/build-service/.tekton/tektoncd-results.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
```
Comment (Michkov) [inline components/build/build-service/.tekton/tektoncd-results.yaml:1]: should not be under build-service but under tekton-results, also the file should be included in kustomization.yaml

Comment (avinal) [reply components/build/build-service/.tekton/tektoncd-results.yaml:1]: Got it thanks

**Comment by avinal** [comment]
Hello @Michkov can you please check if everything is correct?

**Review by Michkov — APPROVED** [review]

**Comment by avinal** [comment]
Is there a problem from tekton-results side?

File: components/build/tekton-results/.tekton/pvc.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: app-studio-default-workspace
```
Comment (adambkaplan) [inline components/build/tekton-results/.tekton/pvc.yaml:5]: Does this need `namespace: build-service`?

**Review by adambkaplan — COMMENTED** [review]
My understanding is this PR does the following:

- Creates Pipelines as Code repo on our AppStudio cluster
- Creates the PVC that can be re-used by Tekton Results builds

Notably, this does _not_ change the deployment to use a HACBS built image. I assume this would come in follow-up PRs that change the `build/tekton-results/kustomization.yaml`.

File: components/build/tekton-results/.tekton/pvc.yaml (hunk 2 above)
Comment (Michkov) [reply components/build/tekton-results/.tekton/pvc.yaml:5]: we are usually running the CI in the same namespace as the application is deployed.

**Pull request merged**